lib/shapes/acqUnit.js

```diff
@@ -1,8 +0,0 @@
-import PropTypes from 'prop-types';
-
-export const acqUnitShape = PropTypes.shape({
```

Comment:
what's wrong with the shape?

Reply:
it's unused now

Reply:
in case you create container, it can be leaved as is

Reply:
it can, but I pass `acqUnitsOptions` from container to filter and use another shape

Reply:
hmm, but if they are not needed - it's ok to remove it